gcp_variant_transforms/transforms/partition_variants_test.py

```diff
@@ -0,0 +1,146 @@
+                              vcfio.Variant(reference_name='chr06', start=0)]
+    # Partition 9
+    expected_partitions[9] = [vcfio.Variant(reference_name='chr10', start=0),
+                              vcfio.Variant(reference_name='  chr10 ', start=0)]
+    # Partition 21
+    expected_partitions[21] = [vcfio.Variant(reference_name='chr22', start=0),
+                               vcfio.Variant(reference_name='chr22  ', start=0)]
+    return expected_partitions
+
+  def _get_nonstandard_variants(self):
+    # All these variants will not match to standard reference_name Reg Exp, thus
+    # they will all end up in partitions >= 22.
+    variants = []
+    # Partition 22
+    variants.append(vcfio.Variant(reference_name='NOT6', start=0))
+    variants.append(vcfio.Variant(reference_name='NOTch3', start=0))
+    # Partition 24
+    variants.append(vcfio.Variant(reference_name='NOTchr1', start=0))
+    variants.append(vcfio.Variant(reference_name='chr99', start=0))
+    # Partition 25
+    variants.append(vcfio.Variant(reference_name='NOT07', start=0))
+    variants.append(vcfio.Variant(reference_name='chr008', start=0))
+    variants.append(vcfio.Variant(reference_name='chr23', start=0))
+    variants.append(vcfio.Variant(reference_name='chrX', start=0))
+    variants.append(vcfio.Variant(reference_name='chrM', start=0))
+    # Partition 26
+    variants.append(vcfio.Variant(reference_name='NOTc4', start=0))
+    variants.append(vcfio.Variant(reference_name='chr0', start=0))
+    variants.append(vcfio.Variant(reference_name='chrY', start=0))
+    return variants
+
+  def _get_nonstandard_variant_partitions(self):
+    # All these variants will not match to standard reference_name Reg Exp, thus
+    # they will all end up in partitions >= 22.
+    expected_partitions = {}
+    # Partition 22
+    expected_partitions[22] = [vcfio.Variant(reference_name='NOT6', start=0),
+                               vcfio.Variant(reference_name='NOTch3', start=0)]
+    # Partition 24
+    expected_partitions[24] = [vcfio.Variant(reference_name='NOTchr1', start=0),
+                               vcfio.Variant(reference_name='chr99', start=0)]
+    # Partition 25
+    expected_partitions[25] = [vcfio.Variant(reference_name='NOT07', start=0),
+                               vcfio.Variant(reference_name='chr008', start=0),
+                               vcfio.Variant(reference_name='chr23', start=0),
+                               vcfio.Variant(reference_name='chrX', start=0),
+                               vcfio.Variant(reference_name='chrM', start=0)]
+    # Partition 26
+    expected_partitions[26] = [vcfio.Variant(reference_name='NOTc4', start=0),
+                               vcfio.Variant(reference_name='chr0', start=0),
+                               vcfio.Variant(reference_name='chrY', start=0)]
+    return expected_partitions
+
+  def test_partition_variants(self):
+    standard_variants = self._get_standard_variants()
```

Comment:
you dont actually need these extra variants; can just get them from `expected_partitions.values()`
you just need to 'flatten' them, which you can do with that for loop i had mentioned previously :)